src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -67,4 +84,4 @@ const InputPassword: UISchemaElementComponent<{
   );
 };
 
-export default InputPassword;
+export default withFormValidationState(InputPassword);
```

Comment:
nit: try replace hoc with hook, should be a bit better performance and consistent coding style